src/ansys/pytwin/evaluate/tbrom.py

```diff
@@ -0,0 +1,270 @@
+import json
+import os
+import struct
+
+import numpy as np
+
+
+class TbRom:
+    """
+    Instantiates a TBROM model part of a TWIN file created by Ansys Twin Builder.
+
+    After a twin model is initialized and its tbrom are instantiated, additional functionalities are available to
+    generate snapshots (in memory or on disk), as well as to project input field data (in case the tbrom is
+    parameterized with an input field).
+
+    Parameters
+    ----------
+    tbrom_name : str
+        Name of the TBROM included in the Twin.
+    tbrom_path : str
+        File path to the TBROM files folder.
+    """
+
+    IN_F_KEY = "binaryInputField"
+    OUT_F_KEY = "binaryOutputField"
+    TBROM_BASIS = "basis.svd"
+    TBROM_SET = "settings.json"
+    TBROM_POINTS = "points.bin"
+
+    def __init__(self, tbrom_name: str, tbrom_path: str):
+        self._tbrom_path = tbrom_path
+        self._tbrom_name = tbrom_name
+        self._infmcs = None
+        self._outmcs = None
+        self._infbasis = None
+        self._outbasis = None
+        self._hasinfmcs = None
+        self._hasoutmcs = False
+
+        files = os.listdir(tbrom_path)
+        infdata = dict()
+        for file in files:
+            if TbRom.IN_F_KEY in file:
+                folder = file.split("_")
+                fname = folder[1]
+                inpath = os.path.join(tbrom_path, file, TbRom.TBROM_BASIS)
+                inbasis = TbRom._read_basis(inpath)
+                infdata.update({fname: inbasis})
+        self._infbasis = infdata
+
+        outpath = os.path.join(tbrom_path, TbRom.OUT_F_KEY, TbRom.TBROM_BASIS)
+        self._outbasis = TbRom._read_basis(outpath)
+        settingspath = os.path.join(tbrom_path, TbRom.OUT_F_KEY, TbRom.TBROM_SET)
+        [nsidslist, dimensionality, outputname, unit] = TbRom._read_settings(settingspath)
+        self._nsidslist = nsidslist
+        self._outdim = int(dimensionality[0])
+        self._outname = outputname
+        self._outunit = unit
+        self._outputfilespath = None
+
+    def points_generation(self, on_disk: bool, output_file_path: str, namedselection: str = None):
+        """
+        Generate a point file either in memory or on disk, for the full field or a specific part. If on disk, the point
+        file is written in the output_file_name path (currently no associated settings file is generated)
+
+        Parameters
+        ----------
+        on_disk: bool
+            Whether the point file is saved on disk (True) or returned in memory (False)
+        output_file_name: str
+            Path where the point file is written if on_disk is True
+        named_selection: str (optional)
+            Named selection on which the point file has to be generated
```

Comment:
```suggestion
        named_selection : str,  optional
            Named selection on which the point file must be generated. The
            default is ``None``.
```
What is the behavior when "None`` is the value? Rewrite the last sentence to something like this:
The default is ``None```, in which case the first named selection is used.